docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -33,3 +33,10 @@ connector:
       Do not change this setting from the default. Non-default values may
       negatively impact performance.
     - 1000
+  * - ``join-pushdown.enabled``
+    - Enable :ref:`join pushdown <join-pushdown>`. Join pushdown is disabled by
+      default. Enabling this may negatively impact performance for some queries.
```

Comment:
Would suggest removing "Join pushdown is disabled by default" since it's saying the same thing as showing the default value to be "false".